web/client/components/geostory/contents/EditorCustomizations/getLinkDecorator.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { openlink } from 'react-draft-wysiwyg';
+
+function findLinkEntities(contentBlock, callback, contentState) {
+    contentBlock.findEntityRanges(
+        (character) => {
+            const entityKey = character.getEntity();
+            return (
+                entityKey !== null &&
+        contentState.getEntity(entityKey).getType() === 'LINK'
+            );
+        },
+        callback,
+    );
+}
+
+function getLinkComponent(config) {
+    const { showOpenOptionOnHover } = config;
+    return class Link extends Component {
+    static propTypes = {
+        entityKey: PropTypes.string.isRequired,
+        children: PropTypes.array,
+        contentState: PropTypes.object
+    };
+
+    state = {
+        showPopOver: false
+    };
+
+    openLink = () => {
+        const { entityKey, contentState } = this.props;
+        const { url } = contentState.getEntity(entityKey).getData();
+        const linkTab = window.open(url, 'blank'); // eslint-disable-line no-undef
+        // linkTab can be null when the window failed to open.
+        if (linkTab) {
+            linkTab.focus();
+        }
+    };
+
+    toggleShowPopOver = () => {
+        const showPopOver = !this.state.showPopOver;
+        this.setState({
+            showPopOver
+        });
+    };
+
+    renderDecorator = () => {
+        const { children, entityKey, contentState } = this.props;
+        const { url, targetOption, attributes } = contentState.getEntity(entityKey).getData();
+        const { showPopOver } = this.state;
+
+        if (attributes && attributes.type === "scrollTo") {
+            return (
+                <span
+                    className="rdw-link-decorator-wrapper"
+                >
+                    <a data-geostory-interaction-type={attributes.type} data-geostory-interaction-params={attributes.params}>{children}</a>
```

Comment:
If we apply the previously suggested changes to attributes, we need also to update this because now attributes contains

 data-geostory-interaction-type and data-geostory-interaction-params